IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/Browser.cs

```diff
@@ -252,5 +266,24 @@ protected IWebDriver InitDriver(BrowserType browser)
             driver.Manage().Timeouts().PageLoad = TimeSpan.FromMinutes(2);
             return driver;
         }
+
+        /// <summary>
+        /// Disposes the current Selenium Driver
+        /// </summary>
+        /// <param name="disposing">Did the call come from Dispose()?</param>
+        protected virtual void Dispose(bool disposing)
+        {
+            if(Disposed)
```

Comment:
```suggestion
            if(_Disposed)
```

Reply:
Oops, missed this when committing your suggestions. Ended up catching it and changing it in a follow-up commit.